package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "@bbc/psammead",
-  "version": "4.0.39",
+  "version": "4.0.40",
   "description": "Core Components Library Developed & Maintained By The Articles and Reach & Languages Team",
   "main": "index.js",
   "private": true,
   "resolutions": {
     "dot-prop": "4.2.1",
-    "prismjs": "1.21.0"
+    "prismjs": "1.22.0",
```

Comment:
Finally letting our old friend `prismjs` tick over to 1.22.0? Or was this unintended?

Reply:
we are indeed letting her tick over. Simorgh was updated last week with the same change